extensions/functions_string.yaml

```diff
@@ -1054,6 +1054,23 @@ scalar_functions:
           - value: "fixedchar<L1>"
             name: "input"
         return: i64
+  -
+    name: char_length_32
+    description: >-
+      Return the number of characters in the input string.  The length includes trailing spaces.
```

Comment:
is `characters` mean code points?

Reply:
Ping @richtia for clarity on what character means. In some system this means bytes. In some system it means utf8 codepoints.